dockertest/subtestbase_unittests.py

```diff
@@ -72,6 +74,36 @@ class DockerTestFail(Exception):
 ]
 
 
+class TestNotRedHat(TestCase):
+
+    def setUp(self):
+        import subtestbase
+        self.subtestbase = subtestbase
+        # Saves some typing
+        self.stbsb = self.subtestbase.SubBase
+        pfx = 'subtestbase_unitest'
+        (self.fd,
```

Comment:
OK, then how about closing immediately after `mkstemp`? 

What bothers me is keeping that fd while all the subsequent code does reopens.